packages/x-components/src/wiring/wiring.types.ts

```diff
@@ -127,8 +129,7 @@ export interface WireService<SomeService> {
    *
    * @param method - The method to invoke.
    * @returns A Wire that expects to receive the function parameter as payload.
-   */
-  <SomeMethod extends PropsWithType<SomeService, MonadicFunction>>(method: SomeMethod): Wire<
+   */ <SomeMethod extends PropsWithType<SomeService, MonadicFunction>>(method: SomeMethod): Wire<
```

Comment:
You're commenting the type, are you?

Reply:
I did this change accidentally when removing unused imports. I rolled it back.